### ISSUE TITLE
Dependency issue with json

### ISSUE THREAD

**Issue by thec0keman** [supremebeing7/time_log_robot#31]
Did a fresh install on Ruby 2.2.5.

Running resulted in:

```
dyld: lazy symbol binding failed: Symbol not found: _rb_str_new_static
  Referenced from: /Users/thec0keman/.gem/ruby/2.2.4/gems/json-1.8.3/lib/json/ext/parser.bundle
  Expected in: flat namespace

dyld: Symbol not found: _rb_str_new_static
  Referenced from: /Users/thec0keman/.gem/ruby/2.2.4/gems/json-1.8.3/lib/json/ext/parser.bundle
  Expected in: flat namespace
```

[This](https://github.com/AtomLinter/linter-rubocop/issues/84) seems to be relevant.  (I manually set my local version to 1.8.2 and it worked).

**Comment by supremebeing7** [comment]
Interesting. I'm not really sure what that's about, since rubocop shouldn't be running except on CI. Also, I am running json 1.8.3 on ruby 2.2.4 and am not getting that. I suppose we could lock `json` at 1.8.2 for now...

**Comment by supremebeing7** [comment]
Hm, doesn't seem to be rubocop-specific, so this is making more sense. It sounds like it might be this: http://stackoverflow.com/a/18474255/3477163. Perhaps give that a try?

**Comment by l0010o0001l** [comment]
![70366051](https://cloud.githubusercontent.com/assets/5507015/17380387/c2c49332-597b-11e6-80bb-3be12663e1dc.jpg)

**Comment by supremebeing7** [comment]
Fixed by #39 (hopefully)
